middleware/idempotency/locker_test.go

```
@@ -59,3 +61,59 @@ func Test_MemoryLock(t *testing.T) {
+	for i := range keys {
+		keys[i] = strconv.Itoa(i)
+	}
+
+	b.Run("UniqueKeys", func(b *testing.B) {
+		lock := idempotency.NewMemoryLock()
+		var keyI atomic.Int32
+		b.RunParallel(func(p *testing.PB) {
+			for p.Next() {
+				i := int(keyI.Add(1)) % len(keys)
+				key := keys[i]
+				lock.Lock(key)
```

golangci-lint (1.62.2)

Error return value of `lock.Lock` is not checked

(errcheck)